File: Javascript/A_NivelIniciante/Exercicio_1036.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
lines = lines[0].split(" ");
A = lines[0];
B = lines[1];
C = lines[2];
var delta = ((B**2)-(4*A*C))
console.log(delta)
if((A!=0) && (delta > 0)){
    console.log("R1 = " + ((-(B) + (delta**(1/2)))/(2*A)).toFixed(5))
    console.log("R2 = " + ((-(B) - (delta**(1/2)))/(2*A)).toFixed(5))
}else{
    console.log("Impossivel calcular")
}
 

File: Python/A_NivelIniciante/Exercicio_1036.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
#url do problema - https://www.urionlinejudge.com.br/judge/pt/problems/view/1036
linha1 = input() 
A, B, C = linha1.split(" ")
A = float(A)
B = float(B)
C = float(C)
delta = ((B**2)-(4*A*C))
if(A!=0) and (delta > 0):
    print("R1 = %.5f" %((-(B) + (delta**(1/2)))/(2*A)))
    print("R2 = %.5f" %((-(B) - (delta**(1/2)))/(2*A)))
else:
    print("Impossivel calcular")
